barretenberg/cpp/src/barretenberg/plonk_honk_shared/composer/permutation_lib.hpp

```diff
@@ -109,13 +110,13 @@ PermutationMapping<Flavor::NUM_WIRES, generalized> compute_permutation_mapping(
 
     // Go through each cycle
     size_t cycle_index = 0;
-    for (auto& copy_cycle : wire_copy_cycles) {
+    for (const auto& copy_cycle : wire_copy_cycles) {
```

Comment:
some changes Facundo suggested (not sure how impactful they are since the compiler is pretty smart usually)